geode-core/src/distributedTest/java/org/apache/geode/cache/ConnectionPoolDUnitTest.java

```diff
@@ -3207,7 +3149,7 @@ public void test028DynamicRegionCreation() {
       assertThat(r).isNotNull();
       String drName = r.getFullPath() + Region.SEPARATOR + dynFromServerName;
 
-      await().pollInterval(100, MILLISECONDS).until(() -> getCache().getRegion(drName) == null);
+      await().until(() -> getCache().getRegion(drName) == null);
```

Comment:
I think this test will run into same thing I hit with another test. There's a brief window in which `getRegion` may throw `RegionDestroyedException` before it starts returning null:
```
await().ignoreException(RegionDestroyedException.class).until(() -> 
    getCache().getRegion(drName) == null);
```

Reply:
done